libnetwork/portallocator/portallocator.go

```diff
@@ -104,20 +88,12 @@ func Get() *PortAllocator {
 	return instance
 }
 
-func getDefaultPortRange() (int, int) {
+func newInstance() *PortAllocator {
 	start, end, err := getDynamicPortRange()
-	if err == nil {
-		start, end, err = sanitizePortRange(start, end)
-	}
 	if err != nil {
-		logrus.WithError(err).Infof("falling back to default port range %d-%d", defaultPortRangeStart, defaultPortRangeEnd)
-		start, end = defaultPortRangeStart, defaultPortRangeEnd
+		logrus.WithError(err).Infof("falling back to default port range %d-%d", DefaultPortRangeStart, DefaultPortRangeEnd)
+		start, end = DefaultPortRangeStart, DefaultPortRangeEnd
```

Comment:
Looks like these consts aren't used outside of the package, so I guess we can keep those